src/eva/data/mon_data_space.py

```diff
@@ -300,52 +302,66 @@ def read_ieee(self, file_name, coords, dims, ndims_used, dims_arr, nvars, vars,
 
         filename = os.path.join(file_path, file_name) if file_path else file_name
 
+        load_data = True
         if not os.path.isfile(filename):
             self.logger.info(f"WARNING:  file {filename} is missing")
-            rtn_array = None
-            return rtn_array
-
-        # read binary Fortran file
-        f = FortranFile(filename, 'r', '>u4')
+            load_data = False
+        else:
+            f = FortranFile(filename, 'r', '>u4')
 
-        if ndims_used == 1:
-            rtn_array = np.empty((0, dims[dims_arr[0]]), float)
+        if ndims_used == 1:		# MinMon
+            rtn_array = np.empty((0, dims[dims_arr[0]]), dtype=float)
+            if not load_data:
+                zarray = np.zeros((dims[dims_arr[0]]), float)
             dimensions = [dims[dims_arr[0]]]
 
-        if ndims_used == 2:
+        if ndims_used == 2:		# RadMon time, bcoef, bcor, OznMon time
             rtn_array = np.empty((0, dims[dims_arr[0]], dims[dims_arr[1]]), float)
+            if not load_data:
+                zarray = np.zeros((dims[dims_arr[0]], dims[dims_arr[1]]), float)
             dimensions = [dims[dims_arr[0]], dims[dims_arr[1]]]
 
-        if ndims_used == 3:
+        if ndims_used == 3:		# RadMon angle
```

Comment:
same comment, I think that might be too many spaces (unless the linter doesn't complain then it's fine)

Reply:
As above linter is good with this.